bin/install-wp-tests.sh

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env bash
```

Comment:
Should we use the latest scaffold tests?  It has a handy `[skip-database-creation]` paramater which I find useful sometimes. 
Latest: https://github.com/wp-cli/scaffold-command/blob/479cd4f10f638d7a46a5b4a60c80bd4b4a6bd5e2/templates/install-wp-tests.sh

Reply:
updated